lasagne/layers/cuda_convnet.py

```diff
@@ -99,7 +99,9 @@ class Conv2DCCLayer(CCLayer):
         An initializer for the weights of the layer. This should initialize the
         layer weights to a 4D array with shape
         ``(num_filters, num_input_channels, filter_rows, filter_columns)``.
+
          If automatic dimshuffling is disabled (see notes), the shape should be
+
         ``(num_input_channels, input_rows, input_columns, num_filters)``
```

Comment:
>> But what exactly does it say?

> I think it is the long [...]

Went to check it myself. The exact error given by Sphinx is "68: ERROR: Unexpected indentation.", and it's because there is an extra space before "If automatic dimshuffling ...". Adding the blank lines worked because it makes the line stand as its own paragraph, so it doesn't have to match the indentation of the other paragraphs. The correct solution would be removing a space character before "If", not inserting blank lines. If you can fix that, I'm fine with merging (and we're down to zero complaints by Sphinx, yay)!